tox.ini

```diff
@@ -18,7 +18,7 @@ minversion=1.8
 [testenv]
 install_command = pip install --ignore-installed {opts} {packages}
 passenv = DISPLAY PYTHONPATH
-setenv = PYTHONPATH = {toxinidir}:
+setenv = PYTHONPATH = {toxinidir}
```

Comment:
apparently this breaks the deps install on appveyor. good news is it seems its not needed anyways since test still run just fine on travis.ci.

Reply:
The colon is os specific: https://tox.readthedocs.io/en/latest/config.html#globally-available-substitutions

But I agree there is no need for it here.